scripts/test_docker_compose.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/expect
```

Comment:
hmm... should we put this into docker/compose/test.sh ?

Reply:
ha that's where I had it originally :-)